source/installation-guide/upgrading/latest_wazuh3_minor.rst

```diff
@@ -77,6 +77,30 @@ Upgrade the Wazuh manager
 
     # apt-get update && apt-get install --only-upgrade wazuh-api
 
+3. Disable the Wazuh repository:
```

Comment:
There's no step to "Enable the Wazuh repository". Any person who follows this guide will have the Wazuh repositories locked (from #493) but currently, there's no step to re-enable it.

Example: [Documentation](https://documentation.wazuh.com/current/installation-guide/upgrading/latest_wazuh3_minor.html#upgrade-to-the-latest-elastic-stack-version)